src/parameters/Parameters.jl

```diff
@@ -44,6 +48,13 @@ Base.@kwdef struct TurbulenceConvectionParameters{FT} <: ATCP
     max_area_limiter_power::FT
 end
 
+Base.@kwdef struct SurfaceTemperatureParameters{FT} <: ASTP
+    SST_mean::FT
+    SST_delta::FT
+    SST_wavelength::FT
+    SST_wavelength_latitude::FT
+end
+
```

Comment:
@nefrathenrici Is it better to group parameters in a struct like this or add them to ClimaAtmosParameters?

Reply:
I grouped them like this so they would be broadcasted correctly into surface_state_to_conditions. ClimaAtmosParameters never gets passed into this function, where surface_temperature is called.

Reply:
That makes sense, thanks!